src/friendly-iframe-embed.js

```diff
@@ -452,6 +454,144 @@ export class FriendlyIframeEmbed {
       this.visibilityObservable_.fire(this.visible_);
     }
   }
+
+  /**
+   * @return {!HTMLBodyElement}
+   * @visibleForTesting
+   */
+  getBodyElement() {
+    return /** @type {!HTMLBodyElement} */ (
+        (this.iframe.contentDocument || this.iframe.contentWindow.document)
+            .body);
+  }
+
+  /**
+   * @return {!./service/vsync-impl.Vsync}
+   * @visibleForTesting
+   */
+  getVsync() {
+    return vsyncFor(this.win);
```

Comment:
The test would have even more boilerplate code otherwise. It would need to `installFriendlyIframeEmbed`, make sure that other resources are installed and FIE is attached to document before being able to mock them. It's more work and turns the test into something more akin to an integration test.

Adding helper methods lets us just short-cut this. It's also not an issue in terms of code size since the compiler will strip out these intermediate methods :)